sound/smooth_swing_cfx_config.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SOUND_SMOOTH_SWING_CFX_CONFIG_H
```

Comment:
Please make a separate pull request for this.

Reply:
pull request #25 opened.